go/exec/exec.go

```diff
@@ -206,6 +217,78 @@ func (e *Exec) Execute() (err error) {
 	return nil
 }
 
+func (e Exec) SendNotification() error {
```

Comment:
nit: Rename to `SendNotificationForRegression`. `SendNotification` seems like a generic function name which can be used to send any notification

Reply:
Fixed via 6caea90